### PR TITLE
ENH keep a masked fraction image

### LINKED ISSUE #169: Record fraction of each pixel that is interpolated

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds an image that indicates the fraction of epochs that were masked for each pixel.

closes #169

### CODE REVIEW

File: pizza_cutter/des_pizza_cutter/_constants.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
 BMASK_CUTOUT_EXTNAME = 'bmask_cutouts'
 ORMASK_CUTOUT_EXTNAME = 'ormask_cutouts'
 NOISE_CUTOUT_EXTNAME = 'noise_cutouts'
+INTERP_CUTOUT_EXTNAME = 'interp_cutouts'
+IMASK_CUTOUT_EXTNAME = 'imask_cutouts'
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_constants.py:25]: @esheldon I am declaring new MEDS extensions `interp` and `imask`. They hold the value of any interpolated flux in the image and the fraction weighted by area of each pixel that is interpolated.

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: Are you ok with these names?

Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: so `interp` means "interpolated flux" and `imask` means fraction of area of each pixel that is interpolated?  The correspondence of name and what they are seems a bit fuzzy to me.

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: OK. Do you have suggestions? I tried and a few times and I got stuck. More than happy to rename!

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: imask is the average of a 0 or 1 for SE image where 1 marks interpolated. So it is not quite the fraction of area of a pixel.

Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: maybe `interp_flux_cutouts` ?

I didn't follow "average of 0 or 1".   But maybe `fracmask_cutouts` ?  I don't know, I guess I don't know what the `i` in `imask` is standing for

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: Ah. So suppose we have an image f_i of either 0 or 1 for each SE in the coadd and we have the weight w_i used for that image in the coadd. Then `imask` is `\sum f_i w_i / \sum w_i`. Does that make sense?

Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: What is `f_i` ?

but don't waste too much time on me, go ahead if you need to

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: f_i is an array that has value 1 if a pixel is interpolated and zero otherwise

Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_constants.py:25]: OK.  As I said, go ahead without me I need to sign off for a while

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_constants.py]: I refactored to something simpler.

**Pull request merged**